packages/core-data/src/selectors.ts

```diff
@@ -18,18 +18,8 @@ import { STORE_NAME } from './name';
 import { getQueriedItems } from './queried-data';
 import { DEFAULT_ENTITY_KEY } from './entities';
 import { getNormalizedCommaSeparable, isRawAttribute } from './utils';
-import type {
-	Context,
-	DefaultContextOf,
-	EntityRecordOf,
-	KeyOf,
-	Kind,
-	KindOf,
-	Name,
-	NameOf,
-	User,
-	WpTemplate,
-} from './entity-types';
+import type { User, WpTemplate } from './entity-types';
+import type * as ET from './entity-types';
```

Comment:
Would it also work to consolidate those types under one name? Example:

```suggestion
import type * as entityTypes from './entity-types';
```